indexer/trie_builder.hpp

```diff
@@ -131,11 +131,11 @@ struct ChildInfo
   size_t GetEdgeSize() const { return m_edge.size(); }
 };
 
-template <typename TValueList>
+template <typename ValueList>
 struct NodeInfo
 {
-  uint64_t m_begPos;
-  TrieChar m_char;
+  uint64_t m_begPos = 0;
```

Comment:
Consider moving it to the bottom while you're here.

Reply:
Done